### PR TITLE
ci: Run accessibility test periodically

### LINKED ISSUE #315: Alternative for circle ci frontend tests
The current Circle CI job runs a frontend test with the following command:
```
docker run --rm -e BASIC_AUTH=$BASIC_AUTH frontend yarn test $endpoint
```

The `$endpoint` can be either `https://www.ietf.org` or `https://wwwdev.ietf.org`.
This should be implemented in a way that it runs frontend tests on a locally hosted container.

### PR DESCRIPTION
**Pull request by kesara**
This change adds a daily GHA that runs accessibility tests on www.ietf.org daily.
Note that www.iab.org is commented out since we need to address a few accessibility issues. See #498.

Fixes #315

### CODE REVIEW

File: .github/workflows/accessibility-test.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: Run accessibility tests
+on:
+  schedule:
+    - cron: '0 0 * * *'
```
Comment (microamp) [inline .github/workflows/accessibility-test.yml:4]: Question: Is the timezone by default UTC?

**Review by microamp — APPROVED** [review]
Looks okay to me. 

Do we want the accessibility test as part of PR checks too?

**Comment by kesara** [comment]
> Looks okay to me.
> 
> Do we want the accessibility test as part of PR checks too?

No, it doesn't make sense now because these tests run on the production site.
To make them able to run per PR (in a meaningful way), we will need to have stub data with docker running them as a web service.

File: .github/workflows/accessibility-test.yml (hunk 1 above)
Comment (kesara) [reply .github/workflows/accessibility-test.yml:4]: Yes, GHA seems to run on UTC by default: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule

**Pull request merged**